src/components/Login/Login.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import Input from '../../common/Input/Input';
+import Button from '../../common/Button/Button';
+import './Login.css';
+
+const Login = ({ setUserName }) => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const navigate = useNavigate();
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    try {
+      const response = await fetch('http://localhost:4000/login', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ email, password }),
+      });
+
+      if (response.ok) {
+        const result = await response.json();
+        console.log('API Response:', result);
```

Comment:
Please remove console.logs from PRs as we treat this as if we're writing production-ready code 😅 (we can keep console.error as they serve a slightly different purpose)